scripts/release.cjs

```diff
@@ -1,7 +1,28 @@
 const fs = require('fs');
+const glob = require('glob');
 const path = require('path');
 const execa = require('execa');
 
+const root = path.resolve(__dirname, '..');
+
+/** Update  */
+function updateLocalDepVersions(dependencyName, newVersion) {
+  const pkgJsonLocs = glob.sync('**/*/package.json', {
+    cwd: root,
+    ignore: ['create-snowpack-app/**/*'],
```

Comment:
Don’t update `create-snowpack-app` templates (unless… should we?)

Reply:
I'd say also update the CSA templates unless we're on a prerelease

Reply:
Hm. Yeah I can add conditional updating based on the tag.